docs/radio.rst

```diff
@@ -0,0 +1,67 @@
+Radio
+*****
+
+.. py:module:: radio
+
+The ``radio`` module allows devices to work together via simple wireless
+networks.
+
+Functions
+=========
+
+.. py:function:: send(message)
+
+    Sends a message, expressed as a string.
+
+.. py:function:: receive()
+
+    Receive the next incoming message on the message queue. Returns ``None`` if
+    there are no pending messages.
```

Comment:
Note that to communicate with devices using DAL-based radio you need to send a few special bytes at the start of the message, like protocol number and version.  These are specific to how the DAL works and have nothing to do with any other nRF device or on-air protocol, so we shouldn't bake them into our implementation.  Rather we should let users just send whatever bytes they like, and they can send the special prefix if they want to talk to other DAL-based devices.  Any data coming from a DAL device will be bytes, not a string.

Also note that I2C, SPI and UART classes all return bytes objects.  But I guess that radio will find more use with younger users than I2C/SPI/UART and so would benefit from a friendlier API.